caramel/config.py

```diff
@@ -71,3 +91,29 @@ def _get_config_value(
             env_var,
         )
     return result
+
+
+def get_log_level(argument_level, root=None, env=None):
```

Comment:
"root" isn't necessary the "root logger" here, just _a_ logger that defaults to the root if None is given.

From the caller of "get_log_level" passing in an argument of "root" does not make sense.